tests/unit/test_database_orm.py

```diff
@@ -0,0 +1,166 @@
+import unittest
+
+from dcpquery.db import Bundle, BundleFileLink, Process, File
+from dcpquery.etl import load_links
+from tests import vx_bundle, vx_bf_links, vx_bundle_aggregate_md, mock_links
+
+from dcpquery import config
+
+
+class TestBundles(unittest.TestCase):
+    def test_insert_select_bundle(self):
+        # config.reset_db_session()
+        config.db_session.add(vx_bundle)
+        config.db_session.commit()
+
+        # select bundle
+        bundle = Bundle.select_bundle(vx_bundle.fqid)
+        res = config.db_session.query(Bundle).filter(Bundle.uuid == vx_bundle.uuid,
+                                                     Bundle.version == vx_bundle.version).all()
+        result = list(res)
+        self.assertEqual(len(result), 1)
+
+        self.assertEqual(result[0], bundle)
+        self.assertEqual(result[0].uuid, vx_bundle.uuid)
+        self.assertEqual(result[0].version, vx_bundle.version)
+        expect_version = vx_bundle.version.strftime("%Y-%m-%dT%H%M%S.%fZ")
+        self.assertEqual(result[0].fqid, f"{vx_bundle.uuid}.{expect_version}")
+        self.assertDictEqual(result[0].aggregate_metadata, vx_bundle_aggregate_md)
+        self.assertEqual(len(result[0].files), len(vx_bundle.files))
+        self.assertSetEqual(set(f.fqid for f in vx_bundle.files), set(f.fqid for f in result[0].files))
+
+    def test_delete_bundles_deletes_list_of_bundles(self):
+        bundle_fqids = [bundle[0] for bundle in config.db_session.query("fqid FROM bundles LIMIT 3;").all()]
```

Comment:
Since this is the ORM test suite, it's probably best to replace all the `query()` queries with their ORM equivalents.

Reply:
I replaced the ones that have ORM equivalents but I only wrote functions for the most common, do you think its worth it to write them for all queries we are using?

Reply:
I was referring to statements like `f"SELECT * FROM bundle_file_links where bundle_fqid='{bundle_fqid}'"`:
- This is a SQL injection vector. Even though the input is trusted here, as a matter of safety habit, never inline variables into SQL statements like this. Instead pass them to the executor as parameters.
- You can express this statement in ORM using something like `dcpquery.config.db_session.execute(select([BundleFileLink]).where(BundleFileLink.bundle_fqid == bundle_fqid)).fetchall()`

For any given test suite, I propose that we either use raw SQL (with parameters passed to psycopg2), or full-on ORM like the above. No need to rewrite existing statements now, but just the new ones you're adding.

Reply:
(You will need `from sqlalchemy.sql import select; from dcpquery.db import BundleFileLink` for the above to work)

Reply:
👍